products/spanner/terraform.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Database: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{instance}}/{{name}}"
+    import_format: ["instances/{{instance}}/databases/{{name}}", "{{project}}/{{instance}}/{{name}}", "{{instance}}/{{name}}"]
```

Comment:
Can we add `projects/{{project}}/instances/{{instance}}/databases/{{name}}`?

Reply:
Sure.